Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM lukemathwalker/cargo-chef:latest-rust-1 AS chef
+WORKDIR /beerus
+
+FROM chef AS planner
+COPY . .
+RUN cargo chef prepare --recipe-path recipe.json
+
+FROM chef AS builder
+COPY --from=planner /beerus/recipe.json recipe.json
+RUN cargo chef cook --release --recipe-path recipe.json
+COPY . .
+RUN cargo build --all --locked --release
+
+FROM debian:buster-slim AS runtime
```

Comment:
We could replace the debian base image by an alpine to get a lighter image, however this would require either to disable cargo chef layer caching, or restricting the target to x86 linux. In the context of the CI I kept a heavier version in order to allow:
- faster builds with cache (25min -> 5min)
- support to build beerus for other architecture (e.g. arm)